Update JuMP-dev 2023 details

odow opened the pull request:
Let me confirm the plan with Ranjan in my call tomorrow morning (12:00 eastern) before we publish this.

Updated form link is https://forms.gle/1pWBhu3VwBUophFV7. Please make/suggest changes/comments.

blegat commented on _posts/2022-11-20-jump_dev_2023.md:
```diff
@@ -19,32 +18,35 @@ practitioners with interests in the methodological, algorithmic, and software as
+#### July 29
 
-The deadline for applications is January 31st.
+ * 0900-1400: hackathon
```
```suggestion
 * 9h-14h: hackathon
```

odow replied:
Ha. Adding an `h` after makes it even more confusing to NZ speakers. `09:00 - 14:00` or `9am - 2pm`?

chriscoey replied:
9am - 2pm is the least pretentious 😄

odow replied:
Is 24-hr time pretentious???

chriscoey replied:
haha it's fine. people in america don't use the `h` Benoit suggested though

blegat replied:
9am - 2pm is fine then :laughing:

blegat commented:
For the question "Do you have any relevant code or academic work that is publicly available? What is your GitHub and/or Julia discourse handle?", what should the participant answer if he has code but no Github handle ?

mlubin commented on _posts/2022-11-20-jump_dev_2023.md:
```diff
@@ -19,32 +19,46 @@ practitioners with interests in the methodological, algorithmic, and software as
-To attend JuMP-dev, you must purchase a ticket to JuliaCon 2023. Ticket sales
-will open in early 2023.
+A preliminary schedule is as follows. The full JuliaCon runs from July 25-29. By
+purchasing a ticket to JuMP-dev, you can also attend the full JuliaCon
```
The wording "purchasing a ticket to JuMP-dev" is a bit confusing given that below we talk about purchasing a ticket to JuliaCon. The tickets are for JuliaCon.

odow replied:
I've changed the sentence.

chriscoey commented:
in my browser, the form https://forms.gle/1pWBhu3VwBUophFV7 seems to lack consistent vertical spacing before the "Value of the awards" section title.

chriscoey commented on _posts/2022-11-20-jump_dev_2023.md:
```diff
@@ -19,32 +19,45 @@ practitioners with interests in the methodological, algorithmic, and software as
+**To give a talk, [submit a proposal](https://forms.gle/1pWBhu3VwBUophFV7)**
 
-The deadline for talk submissions is January 15, 2023.
+The deadline for talk submissions is Feburary 28, 2023.
```
```suggestion
The deadline for talk submissions is February 28, 2023.
```

chriscoey commented on _posts/2022-11-20-jump_dev_2023.md:
```diff
@@ -19,32 +19,45 @@ practitioners with interests in the methodological, algorithmic, and software as
+## How do I attend?
+
+To attend JuMP-dev, you must purchase a ticket to JuliaCon 2023. Ticket sales
+will open in early 2023. Accepted speakers at JuMP-dev will receive a free
```
we don't have an estimated date/month for tickets going on sale?

odow replied:
Nope. I assume in the next couple of weeks. I think they need to co-ordinate with NumFOCUS, which might take a bit of time.

odow commented:
> seems to lack consistent vertical spacing

Fixed. There was a weird formatting thing in the editor. The live preview didn't match the actual preview, so I added an extra blank line.

odow commented on _posts/2022-11-20-jump_dev_2023.md:
```diff
@@ -19,32 +19,45 @@ practitioners with interests in the methodological, algorithmic, and software as
+#### July 29
 
-The deadline for applications is January 31st.
+ * 09:00-14:00: JuMP-dev hackathon
```
```suggestion
 * 9am-2pm: JuMP-dev hackathon
```
